src/main/java/com/qaprosoft/carina/demo/mytest/page/LoginPage.java

```diff
@@ -17,7 +17,7 @@ public LoginPage(WebDriver driver) {
     }
 
     public String loginFailed() {
-        LOGGER.info("Login result [\" + textLoginFail.getText() + \"]");
+        LOGGER.info("Login result [\" + textLoginFail.getText() + ]");
```

Comment:
Use please this example    LOGGER.info("Login result [" + textLoginFail.getText() + "]");